src/components/organisms/EntityDetails/EntityDetailsContent/Settings/utils.ts

```diff
@@ -8,7 +11,11 @@ export const namingMap: {[key in Entity]: string} = {
   tests: 'test',
 };
 
-export const updateRequestsMap: {[key in Entity]: any} = {
+export const updateRequestsMap: {
+  [key in Entity]: UseMutation<
+    MutationDefinition<any, BaseQueryFn<string | FetchArgs, unknown, FetchBaseQueryError>, never, void, string>
+  >;
+} = {
```

Comment:
```suggestion
export const updateRequestsMap: Record<Entity, UseMutation<MutationDefinition<any, any, never, void>> = {
```

or:

```suggestion
import {type useUpdateTestSuiteMutation} from '@services/testSuites';
import {type useUpdateTestMutation} from '@services/tests';

type UpdateMutation = typeof useUpdateTestSuiteMutation | typeof useUpdateTestMutation;

export const updateRequestsMap: Record<Entity, UpdateMutation> = {
```

Alternatively, the first generic of `MutationDefinition` could be based on `Entity`.